client/src/index.ts

```diff
@@ -346,42 +437,91 @@ export class RestRepository<T> extends Repository<T> {
         body[key] = (value as XataRecord).id;
       }
     }
-    const obj = await this.request('POST', `/tables/${this.table}/data`, body);
-    return this.client.initObject(this.table, obj);
+
+    const response = await this.request<{
+      id: string;
+      xata: { version: number };
+    }>('POST', `/tables/${this.table}/data`, body);
+    if (!response) {
+      throw new Error("The server didn't return any data for the query");
+    }
+
+    // TODO: Review this, not sure we are properly initializing the object
+    return this.client.initObject(this.table, response);
+  }
+
+  async createMany(records: T[]): Promise<T[]> {
+    // TODO: Review the id of the records
+
+    const response = await this.request<{
+      recordIDs: string[];
+    }>('POST', `/tables/${this.table}/bulk`, { records });
+    if (!response) {
+      throw new Error("The server didn't return any data for the query");
+    }
+
+    // TODO: Review this, not sure we are properly initializing the object
+    return response.recordIDs.map((record) => this.client.initObject(this.table, { id: record }));
   }
```

Comment:
I implemented the bulk create method. Will also review it with #15 as we're not properly checking the ``id`` of xata record.

Reply:
Nice!